incubator/gitea/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+description: 'Gitea: Go Git Service'
+name: gitea
+version: 1.3.2
+appVersion: 0.0.1
+home: https://gitea.io/
+icon: https://gitea.io/images/favicon.png
+keywords:
+- git
+- gitea
+maintainers:
+- name: Charlie Drage
```

Comment:
Instead of your name can you please put your github username here. That will make it easier for us to contact you in continued maintenance of the chart.